tritium-core/src/main/java/com/palantir/tritium/event/AbstractInvocationEventHandler.java

```diff
@@ -80,9 +80,9 @@ public final boolean isEnabled() {
      * @param context invocation context
      */
     protected final void debugIfNullContext(@Nullable InvocationContext context) {
-        if (context == null) {
+        if (context == null && log.isDebugEnabled()) {
             log.debug(
-                    "{} encountered null metric context, likely due to exception in preInvocation",
+                    "Encountered null metric context, likely due to exception in preInvocation",
```

Comment:
We have a few places in dialogue where we've intentionally left slf4j placeholders for compatibility with non-sls logging formats. I don't have a strong preference, happy to go with whichever style you prefer in Tritium, but I thought I'd share that context in case it's helpful.

Reply:
Everywhere else we're not using placeholders for args, so just wanted to clean these ones up for consistency.

Reply:
👍 sounds good to me